### ISSUE TITLE
Search results page - Background and foreground colors do not have a sufficient contrast ratio. 

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#2800]
**Search results page**

```
Background and foreground colors do not have a sufficient contrast ratio.
Low-contrast text is difficult or impossible for many users to read. Learn more.
Failing Elements
span.d-none.d-lg-inline

Element location: .prev > .d-lg-inline.d-none
Element source: <span class="d-none d-lg-inline"> Previous</span>

Element has insufficient color contrast of 3.94 (foreground color: #7d7d7d, background color: #fafafa, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
```

ACCEPATNCE CRITERIA:

- [ ] Fix color contrast for element previous next
-

**Comment by sandbergja** [comment]
I believe this has been resolved.  This <span> appears on the show page, and now has #000 text on a #F5F5F5 background, a 19.26:1 contrast ratio that passes.
